ginga/trcalc.py

```diff
@@ -320,7 +319,7 @@ def get_scaled_cutout_wdhtdp_view(shp, p1, p2, new_dims):
     x2, y2, z2 = p2
     new_wd, new_ht, new_dp = new_dims
 
-    x1, y1, x2, y2 = int(x1), int(y1), int(x2), int(y2), int(z1), int(z2)
```

Comment:
It is.  Probably because that routine is not normally used for 2D arrays it hasn't been exercised.  I'll port this fix to master and add a test to boot.  Then take it out of this PR.

Reply:
Fixed in master, rebased this PR.